src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/hooks/useCreateAttributeOption.ts

```diff
@@ -0,0 +1,29 @@
+import {useRouter} from '@akeneo-pim-community/legacy-bridge';
+import {useAttributeContext} from '../contexts';
+
+const useCreateAttributeOption = () => {
+    const router = useRouter();
+    const attribute = useAttributeContext();
```

Comment:
If your route is only dependent on the `attribute`, you can avoid to regenerate it each time the hook is called
```suggestion
    const attribute = useAttributeContext();
    const route = useRoute('pim_enrich_attributeoption_create', {
        attributeId: attribute.attributeId.toString(),
     });
```